### PR TITLE
fix(Modal): aside height calculated incorrectly

### PR DESCRIPTION
**Pull request by GiovannaMonti**
### Description

This PR introduces a fix on the Modal component regarding the height of the `aside` element.

The Storybook setup did not contain any CSS reset file; this means that the `box-sizing` property was implicitly set to `content-box` for certain elements. The `aside` element is an example of this: setting its height to 100% would result in it taking its container's height _plus_ its own vertical padding.

##### Modal
  - updated aside element to have `box-sizing: border-box` and height set to 100%; 
  - added CSS reset to storybook preview
  - updated Modal stories to improve the ease of use
 
image

### [IMPORTANT] PR Checklist

- [x] I am aware of standards and conventions adopted in this repository, defined in the [CONTRIBUTING.md file](https://github.com/mia-platform/design-system/blob/main/CONTRIBUTING.md)

#### PR conventions

Please make sure your PR complies with the following rules before submitting it.

- [x] PR title follows the `<type>(<scope>): <subject>` structure
- [x] The PR has been labeled according to the type of changes (e.g. enhancement, new component, bug).

    > **NOTE**  
    > Some labels are used to generate release note entries: you can find the complete mapping between PR labels and release note categories [here](https://github.com/mia-platform/design-system/blob/main/.github/release.yml).  
    For a more detailed overview of PR labels, please refer to the [dedicated CONTRIBUTING section](https://github.com/mia-platform/design-system/blob/main/CONTRIBUTING.md#pull-request-labels).

#### Additional code checks

Based on your changes, some of these checks may not apply. Please make sure to check the relevant items in the list.

- [ ] Changes are covered by tests 
- [x] Changes to components are accessible and documented in the Storybook
- [ ] Typings have been updated
- [ ] New components are exported from the `src/index.ts` file
- [ ] New files include the Apache 2.0 License disclaimer
- [ ] The browser console does not contain errors

### CODE REVIEW

File: .storybook/preview.ts [hunk 1]
```diff
@@ -23,6 +23,8 @@ import docs from "./addons/docs"
 import themeSwitcher from "./addons/theme"
 import withTheme from "./decorators/theme"
 
+import './reset.css'
```
Comment (fredmaggiowski) [inline .storybook/preview.ts:26]: why is this reset needed? can you explain in a comment that remains for the future please?

**Review by fredmaggiowski — APPROVED** [review]
LGTM

File: .storybook/preview.ts (hunk 1 above)
Comment (GiovannaMonti) [reply .storybook/preview.ts:26]: I don't know if an extensive comment explaining this is necessary, since it's a fairly common practice in frontend

**Review by Danielecina — APPROVED** [review]

**Review by ThisIsDemetrio — APPROVED** [review]

**Review by Danielecina — COMMENTED** [review]
I have a doubt. Reset should be imported from ThemeProvider of them.tsx inside storybook decorators, no?

**Comment by GiovannaMonti** [comment]
> I have a doubt. Reset should be imported from ThemeProvider of them.tsx inside storybook decorators, no?

the reset is intended to be applied to storybook stories; here is the recommendation on how to do so: 
https://storybook.js.org/docs/configure/styling-and-css#import-bundled-css-recommended

**Review by fredmaggiowski — APPROVED** [review]
LGTM

**Pull request merged**